api/tests/core/mails/transactional/users/anniversary_beneficiary_test.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+import pytest
+
+import pcapi.core.mails.testing as mails_testing
+from pcapi.core.mails.transactional.users.anniversary_beneficiary import send_newly_eligible_age_18_user_email
+from pcapi.core.testing import override_features
+import pcapi.core.users.factories as users_factories
+
+
+pytestmark = pytest.mark.usefixtures("db_session")
+
+
+class MailjetSendNewlyEligibleUserEmailTest:
+    @override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=False)
+    def test_send_anniversary_email(self):
+        # given
+        user = users_factories.UserFactory(
+            dateOfBirth=(datetime.now() - relativedelta(years=18, days=5)), departementCode="93"
+        )
+
+        # when
+        send_newly_eligible_age_18_user_email(user)
+
+        # then
+        assert len(mails_testing.outbox) == 1  # test number of emails sent
+        assert mails_testing.outbox[0].sent_data["Mj-TemplateID"] == 2030056
+        assert (
+            mails_testing.outbox[0].sent_data["Vars"]["nativeAppLink"][:118]
+            == "https://passcultureapptestauto.page.link/?link=https%3A%2F%2F"
+            "app-native.testing.internal-passculture.app%2Fid-check%3F"
+        )
+        assert "email" in mails_testing.outbox[0].sent_data["Vars"]["nativeAppLink"]
+        assert mails_testing.outbox[0].sent_data["Vars"]["depositAmount"] == 300
+
+
+class SendinblueSendNewlyEligibleUserEmailTest:
+    @override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=True)
+    def test_send_anniversary_email(self):
+        # given
+        user = users_factories.UserFactory(
+            dateOfBirth=(datetime.now() - relativedelta(years=18, days=5)), departementCode="93"
+        )
```

Comment:
Ces précisions ne me semblent pas utiles, car le code ne dépend ni de la date de naissance, ni du département (pour le mail envoyé via SendinBlue).

```suggestion
        user = users_factories.UserFactory()
```